modules/router-store/src/router_selectors.ts

```diff
@@ -59,6 +58,10 @@ export function getSelectors<V>(
     selectRouterState,
     (routerState) => routerState && routerState.url
   );
+  const selectTitle = createSelector(
+    selectCurrentRoute,
+    (routerState) => routerState && routerState.routeConfig?.title
```

Comment:
```suggestion
    (router) => route && route.routeConfig?.title
```

Reply:
Done, sorry as I dont have a better internet I had to do it using web interface, can you squash while merging.